smtp.go

```diff
@@ -182,7 +182,7 @@ func (c *smtpSender) Close() error {
 
 // Stubbed out for tests.
 var (
-	netDialTimeout = net.DialTimeout
+	NetDialTimeout = net.DialTimeout
```

Comment:
This should probably be moved out of this var block so no one's confused later by the `// Stubbed out for tests.` comment.